packages/edit-site/src/components/editor/utils.js

```diff
@@ -1,22 +1,9 @@
 /* CSS properties */
-export const FONT_SIZE = 'font-size';
 export const LINK_COLOR = '--wp--style--color--link';
```

Comment:
Don't we move the link color prop as well?

Reply:
The link color is used on this file and the other file so it makes sense to keep it as is and share it.